irods/session.py

```diff
@@ -320,11 +320,10 @@ def port(self):
 
     @property
     def server_version(self):
-        try:
-            reported_vsn = os.environ.get("PYTHON_IRODSCLIENT_REPORTED_SERVER_VERSION","")
-            return tuple(ast.literal_eval(reported_vsn))
-        except SyntaxError:  # environment variable was malformed, empty, or unset
-            return self.__server_version()
+        reported_vsn = os.environ.get("PYTHON_IRODSCLIENT_REPORTED_SERVER_VERSION","")
+        if reported_vsn:
+            return tuple(ast.literal_eval(reported_vsn)) 
```

Comment:
If `reported_vsn` cannot be parsed because it was set to an invalid value, an exception will be thrown, correct?

Based on the code, I take it this should never fail because the PRC controls that environment variable?

Reply:
This is probably still badly named, the reported_vsn variable, and maybe also the environment variable.  But yes, the env variable is intended only for use in testing the PRC when the current tip-of-main is not yet updated to the next server version to be released.

Reply:
Ah, got it. This is for testing purposes only.

Resolving.